ai_serving/fasttext_clf/fasttext_clf.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: fasttext-clf
+spec:
+  selector:
+    pod: fasttext-clf
+  ports:
+    - protocol: TCP
+      port: 4675
+      targetPort: 4675
+      name: http
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fasttext-clf-up
+  namespace: default
+data:
+  prod_up: |-
+    mkdir /dl; cd /dl; git clone https://prod:$GITPASS@git.jaseci.org/marsninja/jaseci.git;
+    mv jaseci/ai_serving/ /;
+    cd /ai_serving;
+    python3 -m fasttext_clf.fasttext_clf_app;
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: jsuse-git-pass
+type: Opaque
+data:
+  password: UHIwZE4xbmo0
+---
+apiVersion: v1
+data:
+  .dockerconfigjson: eyJhdXRocyI6eyJyZWdpc3RyeS5qYXNlY2kub3JnIjp7InVzZXJuYW1lIjoiamFzZWNpIiwicGFzc3dvcmQiOiJqNHMzYzEiLCJlbWFpbCI6Imphc29uQG1hcnMubmluamEiLCJhdXRoIjoiYW1GelpXTnBPbW8wY3pOak1RPT0ifX19
+kind: Secret
+metadata:
+  name: jsuseregcred
+type: kubernetes.io/dockerconfigjson
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fasttext-clf
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      pod: fasttext-clf
+  template:
+    metadata:
+      labels:
+        pod: fasttext-clf
+      name: fasttext-clf
+    spec:
+      imagePullSecrets:
+        - name: jsuseregcred
+      containers:
+        - name: fasttext-clf
+          imagePullPolicy: IfNotPresent
+          image: fasttext_clf:latest
```

Comment:
Docker image has a naming convention of using dash instead of underscore. Could you change this to be `fasttext-clf` instead?